tools/src/test/java/org/apache/kafka/tools/MetadataQuorumCommandErrorTest.java

```diff
@@ -53,12 +53,14 @@ public void testDescribeOptions() {
 
     @Test
     public void testRelativeTimeMs() {
+        long validEpochMs = Instant.now().minusSeconds(5).toEpochMilli();
+        assertTrue(MetadataQuorumCommand.relativeTimeMs(validEpochMs, "test") >= 0);
         long nowMs = Instant.now().toEpochMilli();
         assertTrue(MetadataQuorumCommand.relativeTimeMs(nowMs, "test") >= 0);
-        long invalidEpochMs = Instant.EPOCH.minus(1, ChronoUnit.DAYS).toEpochMilli();
+        long invalidEpochMs = Instant.EPOCH.minusSeconds(86400).toEpochMilli();
```

Comment:
nit
Why did we change this? `minus(1, ChronoUnit.DAYS)` was much more readable than `minusSeconds(86400)`